Dockerfile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 
-FROM python:3-slim-buster
+FROM python:3.10-bullseye
```

Comment:
It seems like you switched to a different branch completely. In that case can you please test with just `python:bullseye`

Reply:
Hi @ZetaTwo 

It seems like this is the solution.

```
[+] Building 50.7s (13/13) FINISHED                                                                                                                             
 => [internal] load build definition from Dockerfile                                                                                                       0.0s
 => => transferring dockerfile: 841B                                                                                                                       0.0s
 => [internal] load .dockerignore                                                                                                                          0.0s
 => => transferring context: 2B                                                                                                                            0.0s
 => [internal] load metadata for docker.io/library/python:bullseye                                                                                         2.1s
 => [auth] library/python:pull token for registry-1.docker.io                                                                                              0.0s
 => [1/7] FROM docker.io/library/python:bullseye@sha256:21ce92a075cf9c454a936f925e058b4d8fc0cfc7a05b9e877bed4687c51a5658                                  35.9s
 => => resolve docker.io/library/python:bullseye@sha256:21ce92a075cf9c454a936f925e058b4d8fc0cfc7a05b9e877bed4687c51a5658                                   0.0s
 => => sha256:9c5aa3f126bd652d22fcb5fbaea0446609747aa71454fd01a07810b6172e4b78 2.22kB / 2.22kB                                                             0.0s
 => => sha256:8022b074731d9ecee7f4fba79b993920973811dda168bbc08636f18523b90122 53.70MB / 53.70MB                                                          13.1s
 => => sha256:21ce92a075cf9c454a936f925e058b4d8fc0cfc7a05b9e877bed4687c51a5658 1.65kB / 1.65kB                                                             0.0s
 => => sha256:74707af2d4ec537fd45394f8f6a9d93b1cd6b939f2dc942c2c09cdfa1e636102 7.92kB / 7.92kB                                                             0.0s
 => => sha256:7971239fe1d69763272ccc0b2527efa95547d37c53630ed0a71db4e00d3ef964 5.15MB / 5.15MB                                                             0.7s
 => => sha256:26c861b53509d61c37240d2f80efb3a351d2f1d7f4f8e8ec2e5004c1d86af89c 10.87MB / 10.87MB                                                           2.7s
 => => sha256:1714880ecc1c021a5f708f4369f91d3c2c53b998a56d563d0a9aa9be2488d794 54.68MB / 54.68MB                                                          19.4s
 => => sha256:895a945a1f9ba441c2748501c4d46569edfbc2bfbdb9b47d41e753e752247fdc 189.73MB / 189.73MB                                                        28.1s
 => => sha256:2a83fe9e3053a5f52e699f1078c2abeaad849bb81da9068f1aa541bb5673a21f 6.40MB / 6.40MB                                                            14.7s
 => => extracting sha256:8022b074731d9ecee7f4fba79b993920973811dda168bbc08636f18523b90122                                                                  1.6s
 => => sha256:cad84c97de70ce8ac18eaca0f13ed88dd622a1ee17d4f47c88941116b83a8cac 19.93MB / 19.93MB                                                          20.0s
 => => extracting sha256:7971239fe1d69763272ccc0b2527efa95547d37c53630ed0a71db4e00d3ef964                                                                  0.2s
 => => extracting sha256:26c861b53509d61c37240d2f80efb3a351d2f1d7f4f8e8ec2e5004c1d86af89c                                                                  0.2s
 => => sha256:317609cadfd3e8a9eb5393f8aae884de904c93391615b2eb841f17449709e1b5 243B / 243B                                                                19.6s
 => => extracting sha256:1714880ecc1c021a5f708f4369f91d3c2c53b998a56d563d0a9aa9be2488d794                                                                  2.3s
 => => sha256:f756083c502e7d8ae0753f9c828624027c0b16b8d4a203c46929e9606b459ab9 3.08MB / 3.08MB                                                            20.8s
 => => extracting sha256:895a945a1f9ba441c2748501c4d46569edfbc2bfbdb9b47d41e753e752247fdc                                                                  5.7s
 => => extracting sha256:2a83fe9e3053a5f52e699f1078c2abeaad849bb81da9068f1aa541bb5673a21f                                                                  0.2s
 => => extracting sha256:cad84c97de70ce8ac18eaca0f13ed88dd622a1ee17d4f47c88941116b83a8cac                                                                  0.9s
 => => extracting sha256:317609cadfd3e8a9eb5393f8aae884de904c93391615b2eb841f17449709e1b5                                                                  0.0s
 => => extracting sha256:f756083c502e7d8ae0753f9c828624027c0b16b8d4a203c46929e9606b459ab9                                                                  0.2s
 => [internal] load build context                                                                                                                          0.2s
 => => transferring context: 546B                                                                                                                          0.0s
 => [2/7] RUN mkdir /home/sa_scanner                                                                                                                       0.4s
 => [3/7] COPY src/ /home/sa_scanner/                                                                                                                      0.1s
 => [4/7] COPY pyproject.toml /home/sa_scanner/                                                                                                            0.0s
 => [5/7] COPY README.md /home/sa_scanner                                                                                                                  0.0s
 => [6/7] WORKDIR /home/sa_scanner                                                                                                                         0.0s
 => [7/7] RUN pip install .                                                                                                                               11.5s
 => exporting to image                                                                                                                                     0.6s
 => => exporting layers                                                                                                                                    0.6s
 => => writing image sha256:e07896f3533ac8a38bf573d9d8d080033d3346b89fec57a04020eeaba4b6e9eb                                                               0.0s
 => => naming to docker.io/library/gcptest                                                                                                                 0.0
 ```
 
 Once it's merged I close the issue of our re-consideration of version locking.

Reply:
That's great. Unfortunately, I need to ask you to do one final test. Can you test if `python:slim-bullseye` works?

Reply:
```
patrickgerard ~/Documents/Github/gcp_scanner [main] $ docker build . -t gcppt
[+] Building 19.3s (13/13) FINISHED                                                                                                                              
 => [internal] load build definition from Dockerfile                                                                                                        0.0s
 => => transferring dockerfile: 846B                                                                                                                        0.0s
 => [internal] load .dockerignore                                                                                                                           0.0s
 => => transferring context: 2B                                                                                                                             0.0s
 => [internal] load metadata for docker.io/library/python:slim-bullseye                                                                                     2.0s
 => [auth] library/python:pull token for registry-1.docker.io                                                                                               0.0s
 => [1/7] FROM docker.io/library/python:slim-bullseye@sha256:2f749ef90f54fd4b3c77cde78eec23ab5b8199d9ac84e4ced6ae523ef223ef7b                               6.3s
 => => resolve docker.io/library/python:slim-bullseye@sha256:2f749ef90f54fd4b3c77cde78eec23ab5b8199d9ac84e4ced6ae523ef223ef7b                               0.0s
 => => sha256:474f1e977c7dcc3c47ee9fee5d50764763c111f7ce8a0ce2282638fdf6135919 1.37kB / 1.37kB                                                              0.0s
 => => sha256:2704c6131c1c48593359f62312beb65f1173f8cfe8815c92e4fbdc97b493c8dd 6.91kB / 6.91kB                                                              0.0s
 => => sha256:fcdb9667c46b09d1c1d058681ea4a1db41e66bbc1a71d873a0c9da4f7a92947d 30.06MB / 30.06MB                                                            4.2s
 => => sha256:93a67280259393dc61b3b00d448dfe8c71e2fa81bfd4242f464ee813460608c7 1.06MB / 1.06MB                                                              1.0s
 => => sha256:163225f81387d77f1fdf0aa8292f0612e3754ae1039fc52cf73618f7e21cff3e 12.02MB / 12.02MB                                                            3.4s
 => => sha256:2f749ef90f54fd4b3c77cde78eec23ab5b8199d9ac84e4ced6ae523ef223ef7b 1.65kB / 1.65kB                                                              0.0s
 => => sha256:791f7f0a4fe70aa88ffb61f451ba1b46937c04b104e52a260f85be262e906e83 244B / 244B                                                                  1.5s
 => => sha256:e1fcd7e71152298be00c19f748bc3716f84cf31c472cb19b313c6a9b2bd4cbc9 3.37MB / 3.37MB                                                              2.9s
 => => extracting sha256:fcdb9667c46b09d1c1d058681ea4a1db41e66bbc1a71d873a0c9da4f7a92947d                                                                   1.2s
 => => extracting sha256:93a67280259393dc61b3b00d448dfe8c71e2fa81bfd4242f464ee813460608c7                                                                   0.1s
 => => extracting sha256:163225f81387d77f1fdf0aa8292f0612e3754ae1039fc52cf73618f7e21cff3e                                                                   0.3s
 => => extracting sha256:791f7f0a4fe70aa88ffb61f451ba1b46937c04b104e52a260f85be262e906e83                                                                   0.0s
 => => extracting sha256:e1fcd7e71152298be00c19f748bc3716f84cf31c472cb19b313c6a9b2bd4cbc9                                                                   0.1s
 => [internal] load build context                                                                                                                           0.1s
 => => transferring context: 546B                                                                                                                           0.0s
 => [2/7] RUN mkdir /home/sa_scanner                                                                                                                        0.4s
 => [3/7] COPY src/ /home/sa_scanner/                                                                                                                       0.0s
 => [4/7] COPY pyproject.toml /home/sa_scanner/                                                                                                             0.0s
 => [5/7] COPY README.md /home/sa_scanner                                                                                                                   0.0s
 => [6/7] WORKDIR /home/sa_scanner                                                                                                                          0.0s
 => [7/7] RUN pip install .                                                                                                                                 9.7s
 => exporting to image                                                                                                                                      0.6s
 => => exporting layers                                                                                                                                     0.6s
 => => writing image sha256:86853b95e16c46f82e0c0bed5a1b7b8fdcca0545af27116ddd23dbcd4d73f490                                                                0.0s
 => => naming to docker.io/library/gcppt                                                                                                                    0.0s
```

optimized :)